### PR TITLE
docs: update docs

### PR DESCRIPTION
**Pull request by kingcre**
## Description

This PR includes updates in API and CLI docs.

### CODE REVIEW

**Review by crypin — COMMENTED** [review]
1. Is there a reason why `x/liquidamm/spec` part is empty? Will it be added later?
2. I think we need to update the module name used for [this part](https://github.com/crescent-network/crescent/blob/383f79b230930b0de1ca47f12a5153a949a1b9e1/x/liquidstaking/spec/06_hooks.md?plain=1#L14)
3. This is another issue, but it seems necessary to discuss whether the voting power related to liquidamm balance including cre/bre should be added to the liquidstaking gov part

**Comment by kingcre** [comment]
@crypin 

> 1. Is there a reason why `x/liquidamm/spec` part is empty? Will it be added later?

Updated `x/liquidamm/spec`.

> 2. I think we need to update the module name used for [this part](https://github.com/crescent-network/crescent/blob/383f79b230930b0de1ca47f12a5153a949a1b9e1/x/liquidstaking/spec/06_hooks.md?plain=1#L14)

Handled in #201, in [this file](https://github.com/crescent-network/crescent/pull/201/files#diff-6a9a48478adbcfb0dea7780ea0d1402bd6fae61e975794b7b0a97d80cf756bde).

> 3. This is another issue, but it seems necessary to discuss whether the voting power related to liquidamm balance including btoken should be added to the liquidstaking liquid gov part

Handled in #201.

File: x/liquidamm/spec/01_concepts.md [hunk 1]
```diff
@@ -2,67 +2,4 @@
...
-The amount of the rewards depends on the total amount of staked pool coins and the `liquidfarming` module’s staked pool coin, which can be varied during the auction period.
-Therefore, a bidder to place a bid for the auction should be aware of this uncertainty of the rewards amount.
-
-## Bidding for Auction and Winning Bid
-
-A bidder can place a bid with the pool coin, which is the paying coin of the auction.
-A bidder only can place a single bid per auction of a liquid farm.
-The bid amount of the pool coin must be higher than the current winning bid amount that is the highest bid amount of the auction at the moment.
-The bidder placing the bid with the highest amount of the pool coin becomes the winner of the auction and will takes all the accumulated rewards amount at the end of the auction.
+## LiquidAMM
```
Comment (crypin) [inline x/liquidamm/spec/01_concepts.md:5]: Looks still empty of some LiquidAmm parts
- [concepts](https://github.com/crescent-network/crescent/blob/2a7772a28deac148d4b7c4ef9e734bc4da83daee/x/liquidamm/spec/01_concepts.md)
- [state transition](https://github.com/crescent-network/crescent/blob/2a7772a28deac148d4b7c4ef9e734bc4da83daee/x/liquidamm/spec/03_state_transitions.md)
- [begin_block](https://github.com/crescent-network/crescent/blob/2a7772a28deac148d4b7c4ef9e734bc4da83daee/x/liquidamm/spec/05_begin_block.md)
- [event](https://github.com/crescent-network/crescent/blob/2a7772a28deac148d4b7c4ef9e734bc4da83daee/x/liquidamm/spec/06_events.md)

Is it missing or will it be added later in a separate PR?

Comment (kingcre) [reply x/liquidamm/spec/01_concepts.md:5]: I think we should add it later in a separate PR.

Comment (crypin) [reply x/liquidamm/spec/01_concepts.md:5]: If it's not going to proceed right away, it would be good to create an issue or task so that we don't miss it

**Review by crypin — APPROVED** [review]

File: x/liquidamm/spec/01_concepts.md (hunk 1 above)
Comment (kingcre) [reply x/liquidamm/spec/01_concepts.md:5]: Opened #204

**Pull request merged**